pkg/specgenutil/util.go

```diff
@@ -282,6 +287,7 @@ func CreateExitCommandArgs(storageConfig storageTypes.StoreOptions, config *conf
 		"--network-config-dir", config.Network.NetworkConfigDir,
 		"--network-backend", config.Network.NetworkBackend,
 		"--volumepath", config.Engine.VolumePath,
+		transient,
```

Comment:
I don't think boolean flags work like that. It will take the next argument as the command. I did however change this code a bit to make it more in-line with the others.

Reply:
They do work like that, see `podman run --rm=false alpine ls` for instance.

Reply:
Yes, that is what the code does now. But the above proposed code would do `podman run --rm false alpine ls`, which doesn't work.

Reply:
Thanks!